CHANGELOG.md

```diff
@@ -34,7 +51,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 - Fixed a bug in integer validation of arrays that threw an error when an array contained a mix of strings and integers.
 
-- Fixed a bug in JupyterLab >= 4 and Jupyter Notebook >= 7 that caused latex to not render in plotly charts. 
+- Fixed a bug in JupyterLab >= 4 and Jupyter Notebook >= 7 that caused latex to not render in plotly charts.
```

Comment:
LaTeX not latex

Reply:
oh, it looks like that and some others were added to a past release in error, and not the unreleased version, so hadn't seen. will update and move